kibana-reports/public/components/report_definitions/report_trigger/report_trigger.tsx

```diff
@@ -479,7 +479,7 @@ export function ReportTrigger(props: ReportTriggerProps) {
           error={'Invalid cron expression.'}
           labelAppend={
             <EuiText size="xs">
-              <EuiLink href="https://opendistro.github.io/for-elasticsearch-docs/docs/alerting/cron/" target="_blank">
+              <EuiLink href="https://opendistro.github.io/for-elasticsearch-docs/docs/alerting/cron/" target="_blank" external={true}>
```

Comment:
what does this addition attribute do?

Reply:
Adds the `external` icon in the description screenshot